gm4_blast_furnaces/data/blast_furnaces/functions/verify_furnace.mcfunction

```diff
@@ -0,0 +1,9 @@
+# check furnace and store cooktime
+scoreboard players set @s gm4_bf_data -1
+execute if entity @s[y_rotation=0] if block ~ ~ ~1 furnace[facing=south] store result score @s gm4_bf_data run data get block ~ ~ ~1 CookTime
+execute if entity @s[y_rotation=90] if block ~-1 ~ ~ furnace[facing=west] store result score @s gm4_bf_data run data get block ~-1 ~ ~ CookTime
+execute if entity @s[y_rotation=180] if block ~ ~ ~-1 furnace[facing=north] store result score @s gm4_bf_data run data get block ~ ~ ~-1 CookTime
+execute if entity @s[y_rotation=-90] if block ~1 ~ ~ furnace[facing=east] store result score @s gm4_bf_data run data get block ~1 ~ ~ CookTime
```

Comment:
Does the furnace orientation really matter here? If not, you can just remove this function.

Reply:
Unless you want ore tripling, yes

Reply:
Right